pyEpiabm/pyEpiabm/routine/household_forces.py

```diff
@@ -0,0 +1,57 @@
+#
+# Calculate household force of infection based on Covidsim code
+#
+
+from pyEpiabm.core import Person
+
+
+class HouseholdForces:
+    """Class to calculate the infectiousness and susceptibility
+    parameters for the force of infection parameter, within households.
+    """
+    @staticmethod
+    def household_inf(infector: Person, timestep: int):
+        """Calculate the infectiveness of a household.
+
+        :param infector: Infector
+        :type infector: Person
+        :param timestep: Current simulation timestep
+        :type timestep: int
+        :return: Infectiveness parameter of household
+        :rtype: float
+        """
+        return 0.1
+
+    @staticmethod
+    def household_susc(infector: Person, infectee: Person, timestep: int):
+        """Calculate the susceptibility of a household.
+
+        :param infector: Infector
+        :type infector: Person
+        :param infectee: Infectee
+        :type infectee: Person
+        :param timestep: Current simulation timestep
+        :type timestep: int
+        :return: Susceptibility parameter of household
+        :rtype: float
+        """
+        return 0.1
+
+    @staticmethod
+    def household_inf_force(infector: Person, infectee: Person, timestep: int):
```

Comment:
Seeing as this calculates the infectiousness*susceptibility maybe household_FOI would be a better name to avoid confusion with the household_inf one

Reply:
actually household_foi, sorry

Reply:
Yeah I like this, wasn't massively happy with the original name (even if I read _'foi'_ as freedom of information)